libdnf/dnf-sack.cpp

```diff
@@ -2284,13 +2286,9 @@ void dnf_sack_filter_modules(DnfSack *sack, GPtrArray *repos, const char *instal
     excludeQuery.addFilter(HY_PKG_NEVRA_STRICT, HY_EQ, excludeNEVRAsCString.data());
     excludeQuery.queryDifference(includeQuery);
 
-    excludeNamesQuery.addFilter(HY_PKG_NAME, HY_EQ, namesCString.data());
-    excludeNamesQuery.queryDifference(includeQuery);
-
     excludeProvidesQuery.addFilter(HY_PKG_PROVIDES, &nameDeps);
```

Comment:
Add following comment similar to following:
Exclude packages by their Provides. This also excludes packages by name, because packages also provide their %name = %version-%release.